website2/src/components/homePage/ci/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Text } from "../../text";
+import { SampleCode } from "../../sampleCode";
+import { Terminal } from "../../terminal";
+import styles from "./styles.module.css";
+import { Command, TypingText } from "../../typingText";
+
+const CODE = `import * as garn from "https://garn.io/ts/v0.0.14/mod.ts";
+import * as nixpkgs from "https://garn.io/ts/v0.0.14/nixpkgs.ts";
+
+export const backend = garn.haskell
+  .mkHaskellProject({
+    description: "my Haskell backend",
+    executable: "server",
+    compiler: "ghc94",
+    src: ".",
+  })
+  .withDevTools([nixpkgs.hlint])
+  .addCheck("hlint")\`hlint *.hs\`;`;
```

Comment:
FYI these examples have changed (see garn.io) However, it probably can be done as a pass at the end before we merge the redesign since they will likely continue changing

```suggestion
const CODE = `import * as garn from "https://garn.io/ts/v0.0.18/mod.ts";
import * as nixpkgs from "https://garn.io/ts/v0.0.18/nixpkgs.ts";

export const backend = garn.haskell
  .mkHaskellProject({
    description: "my Haskell backend",
    executable: "server",
    compiler: "ghc94",
    src: ".",
  })
  .withDevTools([nixpkgs.hlint])
  .addCheck("hlint", "hlint *.hs");`;
```